t/balancer.t

```diff
@@ -590,3 +590,44 @@ ok
 qr/\[error\] .*? log_by_lua.*? failed to call: API disabled in the current context/
 --- no_error_log
 [alert]
+
```

Comment:
Will you use the `reindex` script to re-format the test file you have edited? Please see

https://github.com/agentzh/old-openresty/blob/master/bin/reindex